pom.xml

```diff
@@ -66,30 +89,30 @@
       <artifactId>guice-multibindings</artifactId>
     </dependency>
     <dependency>
-      <groupId>${jclouds.groupId}</groupId>
+      <groupId>${jclouds.cloudsoft.groupId}</groupId>
       <artifactId>jclouds-core</artifactId>
-      <version>${project.parent.version}</version>
+      <version>${jclouds.cloudsoft.version}</version>
```

Comment:
This is the right version to include in mvn - it is the version of jclouds-core that the `openstack-nova` will pull in.